src/Sylius/Bundle/AdminBundle/Resources/views/Order/Show/_actions.html.twig

```diff
@@ -1,7 +1,8 @@
 {% import '@SyliusUi/Macro/buttons.html.twig' as buttons %}
 
 {% if sm_can(order, 'cancel', 'sylius_order') %}
-    <form action="{{ path('sylius_admin_order_cancel', {'id': order.id} ) }}" method="post">
+    <a href="{{ path('sylius_admin_order_history', {'id': order.id}) }}" class="ui violet labeled icon button" style="float: right;"><i class="history icon"></i>{{ 'sylius.ui.order_history'|trans }}</a>
```

Comment:
This link should be outside of the ``if`` otherwise it will be hidden for cancelled orders. ;)